### DISCUSSION TITLE
Presets

### DISCUSSION THREAD

**Discussion by paulmcauley** [paulmcauley/klassy#102] [Show and tell]
This is a placeholder for sharing Klassy Window Decoration Preset files.

If you upload a preset file, please could you keep it up-to-date with the latest Klassy release, and also provide a screenshot in your comment.

**Comment by Valeria-Fadeeva** [comment]
[Melawy.tar.gz](https://github.com/paulmcauley/klassy/files/13504641/Melawy.tar.gz)

16 and 20 button size

Need the window decorator for KDE 6 (QT6, KF6).

**Comment by CodeWithAlamin** [comment]
Mac OS like title bar buttons with round border and outline. Let me know if you like it 🙂 🩷

[MacOS title bar buttons - Klassy preset.zip](https://github.com/user-attachments/files/17097443/Klassy.Mac.like.round.window.preset.zip)

**Reply by Michael-A-Phillips** [reply]
Don't work says it was created for a older version of Klassy !!

**Comment by evgeniy-harchenko** [comment]
[Qogir](https://github.com/vinceliuice/Qogir-kde) - like preset: [Qogir.klpw.tar.gz](https://github.com/user-attachments/files/27306119/Qogir.klpw.tar.gz)


<img width="1628" height="970" alt="Qogir" src="https://github.com/user-attachments/assets/35a2f983-1216-42ce-8776-25d8516b9a41" />

**Comment by this-username-system-sucks-ass** [comment]
[Breeze-ishv3.1.klpw.tar.gz](https://github.com/user-attachments/files/18498729/Breeze-ishv3.1.klpw.tar.gz)

nearly identical to breeze but with colored borders (accent color, to show focus) and square windows.

edit: ignore version number, that was only for while I was messing with different options so i could keep track of what I had done.

**Comment by AndyHazz** [comment]
[Klassy-Klean.klpw.tar.gz](https://github.com/user-attachments/files/18996307/Klassy-Klean.klpw.tar.gz)

Sharp rectangle traffic light icons, nice spacing.
Coloured window accents on hover.

**Comment by rageblob** [comment]
[Sweet_Twilight.klpw.tar.gz](https://github.com/user-attachments/files/21332117/Sweet_Twilight.klpw.tar.gz)

Inspired by Sweet KDE Plasma theme.
Easy on the eyes, sleek design. Looks very pleasing with the icon set for Sweet KDE.
Mostly transparent title bars, subtle rounded corners, "traffic lights" button colors.

**Comment by SkyShadowHero** [comment]
[DeepinExp.klpw.tar.gz](https://github.com/user-attachments/files/31587979/DeepinExp.klpw.tar.gz)
Inspired by a linux from China called Deepin, which has a different experience from Windows, MacOS and other Linux themes.(Updated in 2026/8/29)
<img width="1095" height="680" alt="deepin" src="https://github.com/user-attachments/assets/1698e8b5-c8f8-4d15-94be-1811ac8ba1f5" />

**Comment by paulmcauley** [comment]
I edited the above post to show a screenshot. Please do something similar. You can also edit your posts to update your .klpw to be compatible with the latest versions.

I may also include some presets in Klassy if they are liked enough by the community.

**Comment by saichiirt** [comment]
[Mac-Likee.klpw (1).tar.gz](https://github.com/user-attachments/files/25279159/Mac-Likee.klpw.1.tar.gz)

MacOS inspired window decorations for klassy 6.5.2. Close, Maximize and Minimize button background colors pre-set already, so changing color-scheme won't affect them.
<img width="1512" height="850" alt="Screenshot_20260209_125255_cropped" src="https://github.com/user-attachments/assets/de84d98a-dbf5-4c00-93bd-7803741bbfbe" />

Update :
02-13-2026 --  (Enabled "Unison Hovering" and "Match titlebar to application colour")

**Reply by paulmcauley** [reply]
You might also want to enable "Unison hovering" under Button Behaviour and "Match titlebar to application colour" under Titlebar.

**Reply by saichiirt** [reply]
Updated 👍
